src/bonds.cpp

```diff
@@ -570,6 +735,68 @@ TEST_CASE("[Faunus] BondData") {
         }
     }
 
+        SUBCASE("HarmonicDihedral") {
+        ParticleVector p_45deg(4, Particle());
+        p_45deg[1].pos = {0.0, 0.0, 0.0};
+        p_45deg[2].pos = {0.0, 0.0, 2.0};
+        p_45deg[0].pos = {5.0, 0.0, 0.0};
+        p_45deg[3].pos = {10.0, 10.0, 2.0};
+
+        ParticleVector p_90deg(p_45deg);
+        p_90deg[3].pos[0] *= 0;
```

Comment:
`.pos[0].setZero()`

Reply:
```C++
p_90deg[3].pos[0].setZero();
```
Returns:
Member reference base type 'Eigen::DenseCoeffsBase<Eigen::Matrix<double, 3, 1, 0>, 1>::Scalar' (aka 'double') is not a structure or union.

Reply:
fixed it.